charts/ceph-csi-rbd/values.yaml

```diff
@@ -217,6 +217,107 @@ topology:
     - failure-domain/region
     - failure-domain/zone
 
+storageClass:
+  # Specifies whether the Storage class should be created
+  create: true
+  name: csi-rbd-sc
+  # String representing a Ceph cluster to provision storage from.
+  # Should be unique across all Ceph clusters in use for provisioning,
+  # cannot be greater than 36 bytes in length, and should remain immutable for
+  # the lifetime of the StorageClass in use.
+  clusterID: <cluster-ID>
+  # Ceph pool into which the RBD image shall be created
+  # eg: pool: replicapool
+  pool: <pool-name>
+  # RBD image features, CSI creates image with image-format 2
+  # CSI RBD currently supports only `layering` feature.
```

Comment:
@Yuggupta27 @nixpanic @Madhu-1 Is this true ?

Reply:
Thanks for pointing it out. Updated.